modules/Broadcast.js

```diff
@@ -64,9 +40,9 @@ class Broadcast extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    invariant(
+    warning(
```

Comment:
You cannot change a `<Broadcast channel>` because `<Broadcast>` doesn't use React's built-in `setState` method to propagate state changes down the hierarchy. Instead, it communicates changes directly to the `<Subscriber>`s in the tree using its `broadcast` object.